### PR TITLE
Club: implementing footer template part

### LINKED ISSUE #6227: club: Templates: Footer template part
Implement the footer template part based on the designs.

Desktop:
image

Mobile:
image

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Club: implementing footer template part

#### Related issue(s):
Fixes: https://github.com/Automattic/themes/issues/6227

### CODE REVIEW

**Comment by madhusudhand** [comment]
Remove `page.html` from the commit. 
And in general it feels not right to review the template without right typography in place.

**Review by madhusudhand — COMMENTED** [review]
Remove `page.html` from the commit. 
And in general it feels not right to review the template without right typography in place.

**Comment by matiasbenedetto** [comment]
> And in general it feels not right to review the template without right typography in place.

You can just copy `theme.json` from `club/fluid-typography` branch and paste it into this branch. Then you will see both changes together.

File: club/parts/footer.html [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+	<!-- /wp:group -->
+	
+	<!-- wp:paragraph -->
+	<p>Powered by WordPress</p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:spacer {"height":"1rem"} -->
+	<div style="height:1rem" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer --></div>
+	<!-- /wp:group -->
```
Comment (mikachan) [inline club/parts/footer.html:29]: The contents of this file should probably be a pattern, as we need to escape and translate all the text content.

**Review by mikachan — COMMENTED** [review]
The site title and address paragraph aren't showing as flex columns for me:

image

They look fine in the editor, so I think this is possibly a GB bug:

image

If I force that content to stack on the front end, the layout is working great for me at both larger and smaller resolutions.

File: club/style.css [hunk 2]
```diff
@@ -113,6 +113,14 @@ GNU General Public License for more details.
     flex-direction: column;
 }
 
+/*
+ * Footer
+ */ 
+
+ footer .wp-block-site-title a {
+	text-decoration: none;
```
Comment (mikachan) [inline club/style.css:121]: According to the design, it looks like this site title should be italic, which I think we'd need to set using CSS.

Comment (matiasbenedetto) [reply club/style.css:121]: We have already added this as a part of another PR: https://github.com/Automattic/themes/pull/6262/files#diff-ae97e73d61d965a9d30bf401f119e47437c0305c922c481000f7fd20aafbfa2eR447

File: club/parts/footer.html (hunk 1 above)
Comment (matiasbenedetto) [reply club/parts/footer.html:29]: I think that each user will fill the footer with their own data and not translate this fake address data included in the theme, so adding the translatable strings seems pointless in this case,

**Comment by matiasbenedetto** [comment]
> The site title and address paragraph aren't showing as flex columns for me:
> They look fine in the editor, so I think this is possibly a GB bug

Yep, it's a Gutenberg bug, I reported it here: https://github.com/WordPress/gutenberg/issues/42938 and submitted a potential fix: https://github.com/WordPress/gutenberg/pull/42939

File: club/parts/footer.html (hunk 1 above)
Comment (mikachan) [reply club/parts/footer.html:29]: Ah yeah I see, that makes sense. I was thinking of the 'Powered by WordPress' text too but we don't tend to make this translatable anymore either.

**Review by mikachan — APPROVED** [review]
This looks good to me!

**Pull request merged**